tests/unit/auth/permissionChecks.js

```diff
@@ -382,17 +382,50 @@ describe('validatePolicyConditions', () => {
             expected: 'Invalid IP address in Conditions',
         },
         {
-            description: 'Should return a relevant error message if the ' +
-            'condition value type does not match the expected type',
+            description: 'Should accept arrays of IPs',
             inputPolicy: {
                 Statement: [{
                     Condition: {
-                        IpAddress: { 'aws:SourceIp': [] }, // IP should be a string, not an array
+                        IpAddress: {
+                            'aws:SourceIp': [
+                                '10.0.11.0/24',
+                                '10.0.1.0/24',
+                            ],
```

Comment:
Can we have a test that checks we properly authorize an ip given a range in the policy? (and on the other hand, deny if it doesn't match the range). As this condition specifically might be used a lot in production, I would feel more confident). It can be done in the Integration test suite.

Reply:
Yes yes can add a test to Integration